src/Models/LinkCheck.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Portable\FilaCms\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class LinkCheck extends Model
+{
+    use HasFactory;
+    use SoftDeletes;
+
+    protected $fillable = [
+        'title',
+        'origin_resource',
+        'edit_url',
+        'url',
+        'status_code',
+        'status_text',
+        'timeout',
+        'batch_id',
+    ];
+
+    public function latestBatch()
+    {
+        $batch = (new LinkCheck())->orderBy('created_at', 'DESC')->limit(1)->first();
+
+        return optional($batch)->batch_id;
```

Comment:
New in PHP id the ?-> which is the same as using the optional. This would look like this:
```return $batch?->batch_id```

Reply:
Yep, I personally use optional for its readability since it screams what it does haha. But yeah that works too and many won't have an issue interpreting ?->